### PR TITLE
[AMSDK-9052] Alpha Mobile Platform extension docs for schema and auto-collected data

### PR DESCRIPTION
**Pull request by kevinlind**
Additional documentation for Alpha Mobile Platform extension
- Schema used in Alpha release
- Auto-collected data in Mobile Platform Extension
- Update Summary.md to link to pages (follow format in PR #209)

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -106,3 +106,9 @@
   * [Set up Project Griffon](beta/project-griffon/set-up-project-griffon.md)
   * [Using Project Griffon](beta/project-griffon/using-project-griffon.md)
 
+## ALPHA
+
+- [Experience Platform Extension](alpha/experience-platform-extension/README.md)
```
Comment (kevinlind) [inline SUMMARY.md]: This 'README.md' is not in this PR, but is being added in PR #209

File: alpha/experience-platform-extension/schema-creation.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+## Sample schema for mobile extension
+
+For Alpha customers of the Adobe Experience Platform Mobile extension, a sample XDM schema is already created for your use. The schema defines a basic set of things you want to send to Adobe Experience Platform if you have products in your application. 
+
+The sample XDM schema for the Experience Platform Mobile extension extends from the [XDM ExperienceEvent]() class, which is a time-series based class and captures the state of the system when an event, or events, occurred. Three mixins are also included in the schema: 
+
+- [ExperienceEvent Application Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-application.schema.md) - data related to the application generating or targeted by the event
+- [ExperienceEvent Environment Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) - data related to the device, location, and surrounding situation
```
[the remark this answers is not in the document]
Comment (kevinlind) [reply alpha/experience-platform-extension/schema-creation.md]: I changed "situation" to "environment". I think environment is better than situation, but if there's a better word let me know. The usage here is "the situation, or environment, at the time the event occurred, such as operating system, mobile carrier, and IP address".

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The Experience Cloud Identifier (ECID)  that is generated by the Adobe Experiece Cloud Mobile SDK is added to the Platform event's `IdentityMap` . If the event contains an `IdentityMap` object, the ECID is added to the `IdentityMap` If the event does not contain the object, a new object is created and is added to the Platform event.
+
+------
+
+
+
+## Device
+
+[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. The following are the device properties the Experience Platform Mobile extension automatically adds to each Platform event.
```
Comment (rekrishn) [inline alpha/experience-platform-extension/auto-collected-data.md]: Please reword the last sentencer as follows:
"Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:" --> please replace the period with a colon.

File: alpha/experience-platform-extension/schema-creation.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+commerce.setOrder(order);
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Product list items
+
+The `ProductListItemsItem` object includes details for a product that was selected by the customer. A list of  `ProductListItemsItem` objects can be used to identity the products with which the customer has interacted.
```
Comment (rekrishn) [inline alpha/experience-platform-extension/schema-creation.md]: Please reword the 2nd sentence to:
"The following `ProductListItemsItem` objects can be..."

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Type of device being tracked. Currently either "phone" or "tablet".
+
+------
+
+
+
+## Environment
+
+[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [ExperienceCloud Environment Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environment at the time of the event.
```
[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/auto-collected-data.md]: Please reword the last sentence as follows:
"Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:" --> please change from a period to a colon at the end."

File: alpha/experience-platform-extension/schema-creation.md [hunk 6]
```diff
@@ -1,39 +1,39 @@
...
+The [Experience Event Commerce Details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-commerce.schema.md) mixin defines a `Commerce` object and a `ProductListItemsItem` object. The `Commerce` object lets you specify which actions are happening in a list of `ProductListItemsItem`s. 
 
 ### Commerce actions
 
-Each action object defined in the `Commerce` object takes a value, typically set to 1, which indicates that action occurred.
+Each action object that is defined in the `Commerce` object takes a value, typically set to 1, which indicates that action occurred.
 
-- CartAbandons - A product list has been identified as no longer accessible (e.g purchasable) by the user.
+- CartAbandons - A product list has been identified as no longer accessible (for example purchasable) by the user.
```
Comment (rekrishn) [inline alpha/experience-platform-extension/schema-creation.md]: please add a comma between "example" and "purchasable".

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+#### Screen height
+
+The number of horizontal pixels of the device's active display in its default orientation.
+
+#### Screen orientation
+
+The current screen orientation, and the options are portrait and landscape.
+
+#### Type
```
[the remark this answers is not in the document]
Comment (rekrishn) [reply alpha/experience-platform-extension/auto-collected-data.md]: @emdobrin, a table is a great idea for both of the sections...provides a break from all of the text too.

**Comment by kevinlind** [comment]
@rekrishn , @emdobrin Thanks for the review and suggestions. I've updated the docs based on the latest feedback and changed the format of auto-collected-data.md to use tables. I also added another section to auto-collected-data to include the two auto-generated properties ID and Timestamp for the Platform Event object.

File: alpha/experience-platform-extension/schema-creation.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+commerce.setOrder(order);
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Product list items
+
+The `ProductListItemsItem` object includes details for a product that was selected by the customer. The following `ProductListItemsItem` objects can be used to identity the products with which the customer has interacted.
```
Comment (emdobrin) [inline alpha/experience-platform-extension/schema-creation.md]: Should this be `for a product that was selected by the user` and `which the user has interacted`?

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 9]
```diff
@@ -16,7 +16,7 @@ The Experience Cloud Identifier (ECID)  that is generated by the Adobe Experiece
 
 ## Device
 
-[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. The following are the device properties the Experience Platform Mobile extension automatically adds to each Platform event.
+[Device](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/device.schema.md) is included in the [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and details data about the mobile device. Here are the device properties the Experience Platform Mobile extension automatically adds to each Platform event:
```
Comment (rekrishn) [inline alpha/experience-platform-extension/auto-collected-data.md]: In the 2nd sentence, please add "that" between "properties" and "the"

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 10]
```diff
@@ -52,7 +52,7 @@ The type of device that is being tracked. The current options are phone and tabl
 
 ## Environment
 
-[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environment at the time of the event. The following are the environment properties the Experience Platform Mobile extension automatically adds to each Platform event.
+[Environment](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/environment.schema.md) is included in the  [Experience Cloud Environment details](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/experienceevent-environment-details.schema.md) mixin and includes information about the surrounding environment at the time of the event. Here are the environment properties the Experience Platform Mobile extension automatically adds to each Platform event:
```
Comment (rekrishn) [inline alpha/experience-platform-extension/auto-collected-data.md]: In the 2nd sentence, please add "that" between "properties" and "the"

File: alpha/experience-platform-extension/schema-creation.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
...
+Experience Data Model (XDM) schemas are a set a rules that define and validate the customer experience data format in Adobe Experience Platform. XDM schemas are composed of one class and zero or more mixins. For more information about XDM Schemas, see [XDM System overview](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/xdm_system/xdm_system_in_experience_platform.md) and [Basics of schema composition](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/schema_composition/schema_composition.md). For information on creating your own schema, see the tutorial on [Create a schema using the Schema Editor](https://www.adobe.io/apis/experienceplatform/home/tutorials/alltutorials.html#!api-specification/markdown/narrative/tutorials/schema_editor_tutorial/schema_editor_tutorial.md).
```
Comment (rekrishn) [inline alpha/experience-platform-extension/schema-creation.md]: - In the 1st sentence, please replace "a set a" with "a set of"
- in the 3rd sentence, please replace "information on" with "information about"

File: alpha/experience-platform-extension/schema-creation.md (hunk 8 above)
Comment (rekrishn) [reply alpha/experience-platform-extension/schema-creation.md]: that works for me too @kevinlind and @emdobrin

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 12]
```diff
@@ -2,9 +2,18 @@
 
 The Experience Platform Mobile extension automatically collects and adds information to each Platform event that is sent to the Adobe Experience Platform. The following sections provide details about the collected information:
 
+## Platform event
+
+The [Experience Event](https://github.com/adobe/xdm/blob/master/docs/reference/context/experienceevent.schema.md) is a time-based record of an event. The XDM Experience Event schema is the top-level class for all Experience events and contains information about the event, device, environment, and the identity of the user. Here are the properties the Experience Platform Mobile extension automatically adds to each Platform event:
```
Comment (rekrishn) [inline alpha/experience-platform-extension/auto-collected-data.md]: In the 2nd sentence, please add "that" between "properties" and "the"

File: alpha/experience-platform-extension/auto-collected-data.md [hunk 13]
```diff
@@ -2,9 +2,18 @@
...
+
+| Property  | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| Timestamp | The time when the event occurred. The timestamp is set when the Platform event is received by the Experience Platform Mobile extension. |
+| ID        | A unique identifier for the time-series event.               |
+
 ## Identity
 
-The [IdentityMap](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/identitymap.schema.md) defines a map that contains end-user identities.
+The [IdentityMap](https://github.com/adobe/xdm/blob/1c22180490558e3c13352fe3e0540cb7e93c69ca/docs/reference/context/identitymap.schema.md) defines a map that contains end-user identities. Here are the identity properties the Experience Platform Mobile extension automatically adds to each Platform event:
```
Comment (rekrishn) [inline alpha/experience-platform-extension/auto-collected-data.md]: In the 2nd sentence, please add "that" between "properties" and "the"

**Comment by emdobrin** [comment]
@kevinlind my changes were merged into experience-platform-staging, make sure you rebase before merging.

**Comment by kevinlind** [comment]
Thanks @rekrishn and @emdobrin! I've pushed the latest changes.

**Pull request merged**